integration/grafana/pbs_prometheus_launch.py

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env python3
+#
+#  Copyright (c) 2015 - 2024 Intel Corporation
+#  SPDX-License-Identifier: BSD-3-Clause
+#
+
+import os
+import sys
+import re
+import datetime
+import subprocess # nosec
+from getpass import getuser
+from argparse import ArgumentParser
+from time import sleep
+from tempfile import NamedTemporaryFile
+
+def get_host_list(jobid):
+    """Get list of hostnames from a PBS jobid
+
+    """
+    jobid = str(jobid)
+    cmd = ['qstat', jobid]
+    is_ready = False
+    while not is_ready:
+        run_ret = subprocess.run(cmd, stdout=subprocess.PIPE, check=True)
+        last_line = run_ret.stdout.decode().split('\n')[-2]
+        line_split = last_line.split()
+        if len(line_split) >= 5 and line_split[4] == 'R':
+            is_ready = True
+        else:
+            sleep(1)
+    cmd = ['qstat', '-x', '-f', '-w', jobid]
+    run_ret = subprocess.run(cmd, stdout=subprocess.PIPE, check=True)
+    host_str = None
+    for line in run_ret.stdout.decode().split('\n'):
+        if 'exec_host' in line:
+            line_split = line.split()
+            if len(line_split) >= 3:
+                host_str = line_split[2]
+                break
+    if host_str is None:
+        raise RuntimeError('Failed to parse host list from qstat output')
+    hosts =[xx[:xx.find('/')] for xx in host_str.split('+')]
+    hosts = list(set(hosts))
+    hosts.sort()
```

Comment:
```suggestion
    hosts = sorted({xx[:xx.find('/')] for xx in host_str.split('+')})
```
Can do set->sorted list instead of list->set->list->sorted